go/vt/servenv/status.go

```diff
@@ -337,4 +338,8 @@ func init() {
 	if err != nil {
 		log.Exitf("os.Hostname: %v", err)
 	}
+
+	_ = stats.NewGaugeFunc("Uptime", "Uptime in nanoseconds", func() int64 {
+		return int64(time.Since(serverStart).Nanoseconds())
+	})
```

Comment:
@mattlord sounds good 👍!

In the process of moving it to `servenv.go` I noticed a duplicate `time.Time` representing the start time of the server, so I updated `status.go` to use `GetInitStartTime()` instead, let me know what you think

Reply:
technically, this is a different time, since `serverStart` would be `time.Now()` when servenv was _imported_ (roughly, depending on exactly when that line in the `var` block was evaluated), now it's when `Init` was _called_ by some other package

so, i don't particularly care either way, just wanted to call out the difference so you were aware and can pick the one you want/need for your use case

Reply:
@ajm188 that's a good point, those times should be very similar but not the same 👍

I guess if we used a unit as exact as nanoseconds we should be pretty accurate, so I'll restore the `serverStart` time